src/test/java/playSongTests.java

```diff
@@ -0,0 +1,28 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+
+
+public class playSongTests extends BaseTest {
+    @Test
+    public void playSong() throws InterruptedException {
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/223034999-a3f5b295-0cbe-4b83-a820-35f9dd416efe.png)

Please, **delete** `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.